hardhat.config.ts

```diff
@@ -122,6 +126,10 @@ const config: HardhatUserConfig = {
             134: '0xA0C26578F762a06c14A8153F87D0EAA2fBd036af', // Bellecour & local fork
         },
     },
+    // gas reporter
+    gasReporter: {
```

Comment:
Is all ok to always enable it (@iExecBlockchainComputing/team-blockchain )? (quite noisy in dev with big table to scroll while debugging, maybe enable it only with `REPORT_GAS=true`)